doc/api/errors.md

```diff
@@ -1047,6 +1047,11 @@ An invalid value has been specified for an HTTP/2 setting.
 
 An operation was performed on a stream that had already been destroyed.
 
+<a id="ERR_HTTP2_INVALID_TOKEN"></a>
+### `ERR_HTTP2_INVALID_TOKEN`
```

Comment:
Introducing a separate error code is not necessary here. The existing `ERR_INVALID_HTTP_TOKEN` should be reused as there is no difference between how token in defined for HTTP/1 and /2.

Reply:
Done.